app/javascript/lib/cell-faceting.js

```diff
@@ -17,7 +17,7 @@ const CELL_TYPE_REGEX = new RegExp(/cell.*type/i)
 
 // Detect if a string mentions disease, sickness, malignant or malignancy,
 // indication, a frequent suffix of disease names, or a common suffix of cancer names
-const DISEASE_REGEX = new RegExp(/(disease|sick|malignan|indicat|itis|osis|oma)/i)
+const DISEASE_REGEX = new RegExp(/(disease|sick|malignan|indicat|itis|isis|osis|oma|medical)/i)
```

Comment:
If you feel like adding it, "syndrome" is another string that would ID potential medically-relevant filters.

Reply:
Good idea -- done in 5668d3c.